.commitlintrc.json

```diff
@@ -0,0 +1,38 @@
+{
+  "extends": [
+    "@commitlint/config-conventional"
+  ],
+  "rules": {
+    "type-enum": [
+      2,
+      "always",
+      [
+        "docs",
+        "resources",
+        "planner",
+        "config",
+        "ci",
+        "api",
+        "workflows",
+        "controllers",
+        "tests",
+	"test utils",
+
+        "tests/integration",
+	"integration tests",
```

Comment:
> That's probably because @obnoxxx is basing this off a habit I have of "rewording" directory paths to common English when possible. So "tests/integtration" can become "integration tests" or "tests/utils" becomes "test utils" or (hypothetically) "modules/password" might become "password module". We can debate the wisdom of this separately but I'm pretty sure this is the reason.

exactly 😄